submissions/DmytryjK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Hooli Popup</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.png">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="container">
+            <a class="header__logo-link" href="#">
+                <img class="header__logo-img" src="images/logo.svg" alt="site logotype">
+            </a>
+            <nav class="header__navigation">
+                <ul class="header__list">
+                    <li class="header__items">
+                        <a class="header__links" href="#">Mail</a>
+                    </li>
+                    <li class="header__items">
+                        <a class="header__links" href="#">Images</a>
+                    </li>
+                    <li class="header__items">
+                        <label class="header__popup-label" for="checkbox">
+                            <input class="header__popup-input" id="checkbox" type="checkbox">
+                            <span class="header__popup-btn-img"></span>
+                            <ul class="header__popup-submenu">
+                                <li class="header__popup-item">
+                                    <a class="header__popup-link" href="#">
+                                        <picture>
+                                            <source srcset="images/WEBP/calendar.webp" type="image/webp">
+                                            <source srcset="images/calendar.png" type="image/jpeg"> 
+                                            <img class="header__popup-img" src="images/calendar.png" alt="calendar icon">
```

Comment:
Read [this article](https://htmlacademy.ru/blog/articles/alt-text) about `alt` attribute.
Sorry for russian but it's a good one.